clarifai/cli/deployment.py

```diff
@@ -0,0 +1,78 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.nodepool import Nodepool
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group(['deployment', 'dpl'])
+def deployment():
+  """Manage Deployments: create, delete, list"""
+  pass
+
+
+@deployment.command()
+@click.option(
+    '-np_id',
+    '--nodepool_id',
+    required=True,
+    help='Nodepool ID for the Nodepool to interact with.')
```

Comment:
I'm not sure if this `--nodepool_id` flag should always to required, because `nodepool_id` info might be present in config passed, one thing we could do is if `--nodepool_id` flag is not passed then fetch it from config if present and if not present in config too then throw nice error by mentioning to pass  nodepool_id either as flag or in config file

I'm not sure if the `--nodepool_id` flag should always be required, as the `nodepool_id` might already be present in the provided config file. One approach could be to check if the `--nodepool_id` flag is omitted, then attempt to retrieve it from the config file if available. If it's not found in the config either, we can display a clear error message instructing the user to specify the `nodepool_id` either as a flag or within the config file. WDYT?

Same for other deployment and nodepool functions

Reply:
Done